src/commands/create.ts

```diff
@@ -74,13 +74,50 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     }
 
     // now that we have parsed the command args, run the create operation
-    const { database: db, ens } = await setupCommand(argv);
+    const { database: db, ens, normalize } = await setupCommand(argv);
 
-    if (argv.enableEnsExperiment && ens)
+    if (argv.enableEnsExperiment && ens) {
       statement = await ens.resolve(statement);
+    }
+    statement = statement
+      .replace(/\n/g, "")
+      .replace(/\r/g, "")
+      .replace(/\s+/g, " ")
+      .trim();
+
+    // Parse the statement to see if more than one table is affected.
+    // If yes, then combine into separate statements and batch.
+    // If no, then process via single statement.
+    const statements = statement.split(";").filter((stmt) => stmt.trim());
+    const normalized = await Promise.all(
```

Comment:
We have to do some parsing in the CLI before we normalize because the go sqlparser will error if you try to pass a single string with multiple create queries.